runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractQueuedStreamView.java

```diff
@@ -29,10 +15,19 @@
 import org.corfudb.runtime.exceptions.TrimmedException;
 import org.corfudb.runtime.object.transactions.TransactionalContext;
 import org.corfudb.runtime.view.Address;
-import org.corfudb.runtime.view.RuntimeLayout;
-import org.corfudb.runtime.view.replication.ChainReplicationProtocol;
 import org.corfudb.util.Utils;
 
+import javax.annotation.Nonnull;
```

Comment:
Lexicographically order imports.

Reply:
This is Intellij's auto import optimization. As you can see other files are also placing `import javax.*`  before `import java.*`